### PR TITLE
Typo "Teams Room Pro"→"Teams Rooms Pro"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://learn.microsoft.com/en-us/microsoftteams/rooms/meetings-with-sip-h323-devices 
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/public/Teams/rooms/meetings-with-SIP-H323-devices.md 
#PingMSFTDocs

### CODE REVIEW

File: Teams/rooms/meetings-with-SIP-H323-devices.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ If you have multiple Teams Rooms all with a Pro licenses deployed in your organi
 
 To enable your Teams Rooms on Windows to use SIP and H.323 dialing:
 
-1. The Teams Room account that is used to sign in to the device must have a Teams Room Pro license assigned.
+1. The Teams Room account that is used to sign in to the device must have a Teams Rooms Pro license assigned.
```
Comment (JohanFreelancer9) [inline Teams/rooms/meetings-with-SIP-H323-devices.md:38]:
```suggestion
1. The Teams Rooms account that is used to sign in to the device must have a Teams Rooms Pro license assigned.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hyoshioka0128, kindly commit the suggested additional change. Many thanks.

**Comment by hyoshioka0128** [comment]
@JohanFreelancer9 - Thanks for the suggestion.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tonysmit 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**